src/amm_core/amm.cairo

```diff
@@ -639,5 +639,9 @@ mod AMM {
         ) -> (Fixed, Fixed) {
             View::get_total_premia(option, position_size, is_closing)
         }
+
+        fn get_fees_percentage(self: @ContractState) -> u128 {
```

Comment:
Shouldn't there also be a setter, ie. Owner can call `set_fees_percentage` so that we don't need to upgrade to change this value?